teams/project-edition-2021.toml

```diff
@@ -1,17 +1,10 @@
 name = "project-edition-2021"
-subteam-of = "core"
+subteam-of = "leadership-council"
```

Comment:
Should we really be modifying this team? It feels like it should be archived instead of modified, almost feels like changing history :)